src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -354,3 +356,21 @@ def get_dataset_name_version(run, dataset_input_name):
     aid = get_run_input_assets(run)[dataset_input_name]["asset"]["assetId"]
     ainfo = get_asset_information(aid)
     return f'{ainfo["assetname"]}:{ainfo["assetversion"]}'
+
+
+def feature_metadata_to_dict(feature_metadata):
+    if  isinstance(feature_metadata, FeatureMetadata): 
+        ret = feature_metadata.__dict__
+        type_name = type(feature_metadata).__name__
+        ret[data_type] = type_name
+        return ret
+    else:
+        return
```

Comment:
nit: return explicitly None here? Maybe change ret variable to be more descriptive.  Also, remove the else, so just:
```
if isinstance(...):
   ...
   return meta_dict
return None
```

Reply:
updated! thanks